scripts/rollup/rollup.config.js

```diff
@@ -60,6 +60,7 @@ function sharedPlugins() {
     return [
         typescript({
             tsconfig: path.join(packageRoot, 'tsconfig.json'),
+            exclude: ['**/__tests__/**'],
```

Comment:
Right, because we still need typechecking in `__tests__`, we just don't want to do anything with it during the Rollup step.